app/templates/direct-award/results.html

```diff
@@ -71,11 +71,32 @@
           }
       ]
       %}
-          {% include "toolkit/documents.html" %}
+        <ul class='document-list'>
+          {% for item in items %}
+            <li class='document-list-item'>
+              {% if item.file_type %}
+                <a href="{{ item.link }}" class='document-link-with-icon' {% if item.download %}download {% endif %}
+                  data-analytics="trackEvent"
+                  data-analytics-category="Direct Award"
+                  data-analytics-action="Download search results format"
+                  data-analytics-label="{{ item.file_type }}"
+                >
+                  <span class='document-icon'>{{ item.file_type }}<span> document:</span></span>
```

Comment:
what's the extra ```span``` here for?

Reply:
this was from the template, https://github.com/alphagov/digitalmarketplace-frontend-toolkit/blob/master/toolkit/templates/documents.html

Should we just update the toolkit instead with the GA options?

Reply:
I didn't update it, just in case it was used in other places (ie. not in Digital Marketplace)

Reply:
Nothing other than the Digital Marketplace frontends should be using our toolkit.

We ought to be using/updating our toolkit things rather than duplicating them out, if they seem like good patterns and we aren't changing them in significant ways. Do have to check which of our apps are using them and where, before making changes tho.

As to whether the span should be there or not: no unless you can find some CSS that relies on it. (In which case I'd argue it still needs a `class`.)